sklearn/metrics/scorer.py

```diff
@@ -519,6 +526,9 @@ def make_scorer(score_func, greater_is_better=True, needs_proba=False,
                neg_mean_squared_error=neg_mean_squared_error_scorer,
                neg_mean_squared_log_error=neg_mean_squared_log_error_scorer,
                accuracy=accuracy_scorer, roc_auc=roc_auc_scorer,
+               roc_auc_ovo=roc_auc_ovo_scorer,
```

Comment:
I still don't know whether it's useful to add such diverse scorers.

Reply:
Maybe it would be good to only define the `roc_auc_ovo_scorer` scorer?

Reply:
The default for `roc_auc_score` is `multiclass="ovo"`, thus `ovr` would be covered by normal `roc_auc_score`. Even with that, it would be good to add a `roc_auc_ovr_scorer`, since "explicit is better than implicit".

Reply:
How did we pick the default again? I usually feel OVR is easier to grasp and it would make the default multi-class behavior be consistent with the multi-label behavior.

I think we should have macro OVR as a scorer as it's quite common. It's probably the default metric I would pick for multi-class classification.